sycl/test-e2e/Basic/accessor/empty_zero_dim_accessor.cpp

```diff
@@ -1,9 +1,7 @@
+// REQUIRES-INTEL-DRIVER: lin: 26690
```

Comment:
Should we enable it for Windows too?

Reply:
I've gone with the newest windows version. It should be sufficient with our CI.